ui/packages/app/web/package.json

```diff
@@ -23,31 +23,58 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@headlessui/react": "^1.4.3",
+    "@headlessui/react": "^1.4.2",
     "@parca/client": "^0.9.1",
     "@parca/dynamicsize": "^0.9.0",
     "@parca/functions": "^0.9.2",
     "@parca/icons": "^0.9.2",
     "@parca/parser": "^0.9.0",
     "@parca/profile": "^0.9.2",
+    "@reduxjs/toolkit": "^1.7.2",
+    "@storybook/addon-actions": "^6.4.15",
```

Comment:
A lot of these look like it belongs to `devDependencies`. I guess, it got added accidentally?

Reply:
Yes, you're right! I think I can safely move those to devDependencies 🙂